js/components/controlBar.js

```diff
@@ -315,8 +317,9 @@ var ControlBar = React.createClass({
     var selectedStyle = {};
     selectedStyle["color"] = this.props.skinConfig.general.accentColor ? this.props.skinConfig.general.accentColor : null;
 
+    var isTooltipEnabled = this.props.skinConfig.controlBar.tooltips ? this.props.skinConfig.controlBar.tooltips.enabled : false;
     var controlItemTemplates = {
-      "playPause": <button className="oo-play-pause oo-control-bar-item"
+      "playPause": (function(alignment){return  <button className="oo-play-pause oo-control-bar-item"
```

Comment:
The 'alignment' parameter is sent to the Tooltip component as a property.
The 'alignment' property inside the Tooltip component is used to define the position of the tooltip 'pointer' i.e. the little triangle at the bottom of the tooltip box. The triangle can be positioned at the left,right or center of the tooltip box.